src/main/java/com/researchspace/dataverse/api/v1/UsersOperations.java

```diff
@@ -29,19 +29,18 @@ public interface UsersOperations {
 
     /**
      * Get token expiration date.
-     * @return java.util.Date token expiration date.
+     * @return java.time.LocalDateTime token expiration.
```

Comment:
This will break existing client code that expects java.util.Date. To avoid doing that, suggest creating a new method with a variant name e.g. 'getTokenExpirationLocalDateTime'